tools/config.py

```diff
@@ -64,7 +64,7 @@ def normalize_config_settings():
 
   # EM_CONFIG stuff
   if not JS_ENGINES:
-    JS_ENGINES = [NODE_JS]
+    JS_ENGINES = [NODE_JS_TEST if NODE_JS_TEST else NODE_JS]
```

Comment:
Can you move these 2 lines to `test/common.py` after the assignment to `NODE_JS_TEST` (then you don't need the embedded if/else here).

Reply:
Oh I see, I still need the ternary, I misunderstood the move was intended to avoid that.

Reply:
I reverted back to the original form, that helped the mypy type check, that insisted thinking that `config.NODE_JS` is `Optional[Any]` and not `List[Str]`